### PR TITLE
Block Editor: Add missing @param docs

### PR DESCRIPTION
**Pull request by mkaz**
## Description

Updates three components in block-editor fixing missing @param docs

See: #22907

## How has this been tested?

Test running:

`npm run lint-js packages/block-editor/src/components/color-style-selector/index.js packages/block-editor/src/components/inserter/index.native.js packages/block-editor/src/components/writing-flow/index.js`

Previous warnings:

## Types of changes

JSDoc changes.

### CODE REVIEW

File: packages/block-editor/README.md [hunk 1]
```diff
@@ -616,6 +616,11 @@ _Returns_
 Handles selection and navigation across blocks. This component should be
 wrapped around BlockList.
 
+_Parameters_
+
+-   _props_ `Object`: Component properties.
+-   _props.children_ `WPElement`: Children to be rendered
```
Comment (ocean90) [inline packages/block-editor/README.md:622]:
```suggestion
-   _props.children_ `WPElement`: Children to be rendered.
```

File: packages/block-editor/src/components/color-style-selector/index.js [hunk 2]
```diff
@@ -39,7 +42,10 @@ const ColorSelectorIcon = ( { style, className } ) => {
 /**
  * Renders the Colors Selector Toolbar with the icon button.
  *
- * @param {Object} colorControlProps colorControl properties.
+ * @param {Object} props                 Component properties.
+ * @param {Object} props.TextColor       Text color component that wraps icon.
+ * @param {Object} props.BackgroundColor Background color componnet that wraps icon.
```
Comment (ocean90) [inline packages/block-editor/src/components/color-style-selector/index.js:47]:
```suggestion
 * @param {Object} props.BackgroundColor Background color component that wraps icon.
```

File: packages/block-editor/src/components/writing-flow/index.js [hunk 3]
```diff
@@ -239,6 +239,9 @@ function selector( select ) {
 /**
  * Handles selection and navigation across blocks. This component should be
  * wrapped around BlockList.
+ *
+ * @param {Object}    props          Component properties.
+ * @param {WPElement} props.children Children to be rendered
```
Comment (ocean90) [inline packages/block-editor/src/components/writing-flow/index.js:244]:
```suggestion
 * @param {WPElement} props.children Children to be rendered.
```

**Review by ocean90 — CHANGES_REQUESTED** [review]

**Review by ocean90 — APPROVED** [review]

**Comment by youknowriad** [comment]
(For the record, I'm changing all the type labels to make sure these PRs should in the same "Documentation" section when the changelog in generated)

**Pull request merged**